app/generator/styles.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   header1: {
```

Comment:
What's the knock-on effect of these changes for statements?

Reply:
Separated common files that required changes, also, common styles that required changes have been separated.